src/pages/api/autocomplete.ts

```diff
@@ -0,0 +1,52 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { z, ZodError } from "zod";
+import { MasterItem, List, Prisma, PrismaClient } from "@prisma/client";
+var stringSimilarity = require("string-similarity");
+
+export function sortByRating(input: Array<{ target: string; rating: number }>) {
+  const sorted = [...input];
```

Comment:
this is the same as writing `const sorted = input;`

Reply:
or are you deep-copying here?

Reply:
We want to create a copy of the input array.